src/ansys/fluent/parametric/local/__init__.py

```diff
@@ -0,0 +1,310 @@
+"""
+Classes for locally defining a parametric study for Fluent without
+running Fluent. The study can then be submitted to be executed in 
+parallel.
+
+Example
+-------
+
+Set up a local study
+
+>>> from ansys.fluent.parametric.local.local import LocalParametricStudy
+>>> local_study = LocalParametricStudy(case_filepath="E:/elbow1_param.cas.h5")
+>>> design_point = local_study.design_point("Base DP")
+>>> design_point.input_parameters['v1'] = 0.0
+>>> for idx in range(1, 20):
+>>>   design_point = local_study.add_design_point("dp_"+str(idx))
+>>>   design_point.input_parameters['v1'] = float(idx)/10.0
+
+Run in Fluent
+
+>>> local_study.run_in_fluent(5)
+
+Display results
+
+>>> for design_point in local_study.design_point_table:
+>>>   for k, v in design_point.input_parameters.items():
+>>>     print("input parameter", k, v)
+>>>   for k, v in design_point.output_parameters.items():
+>>>     print("output parameter", k, v)
+>>> print(72 * "-")
+
+"""
+
+from math import ceil
+from typing import Any, Dict, Union
+
+from ansys.fluent.core.utils.async_execution import asynchronous
+
+from ansys.fluent.parametric import (
+    BASE_DP_NAME,
+    ParametricSession,
+    ParametricSessionLauncher,
+)
+
+try:
+    from ansys.fluent.core.filereader.casereader import CaseReader
+except ImportError:
+    from ansys.fluent.parametric.local.filereader.casereader import CaseReader
+
+
+def convert_design_point_parameter_units(
+    value: Dict[str, Union[float, str]]
```

Comment:
```suggestion
    value: Dict[str, Union[float, int, str]]
```

Reply:
thanks @ajain-work, I wasn't dealing with int originally in the code